Action_inputs.md

```diff
@@ -0,0 +1,135 @@
+  </tr>
+  <tr>
+    <td><code>k8s-secret</code><br/>Secret</td>
+    <td>(Relevant for service account approach) Secret associated with the service account to be used for deployments</td>
+  </tr>
+</table>
+
+### Inputs specific to create secret (Optional)
+**Note:** Below mentioned inputs are only required when creating new secrets. For creating new secret please provide `secret-name` and required inputs. For skipping this part (create secret), do not specify `secret-name` as input.
+
+<table>
+  <thead>
+    <tr>
+      <th>Action inputs</th>
+      <th>Description</th>
+    </tr>
+  </thead>
+  <tr>
+    <td><code>container-registry-url</code><br/>Container registry url</td>
+    <td>(Only required when creating a new secret) Container registry url. </td>
+  </tr>
+  <tr>
+    <td><code>container-registry-username</code><br/>username</td>
+    <td>(Only required when creating a new secret) Container registry username</td>
+  </tr>
+  <tr>
+    <td><code>container-registry-password</code><br/>password</td>
+    <td>(Only required when creating a new secret) Container registry password</td>
+  </tr>
+  <tr>
+    <td><code>container-registry-email</code><br/>Email</td>
+    <td>(Optional) Container registry email</td>
+  </tr>
+  <tr>
+    <td><code>secret-type</code><br/>secret type</td>
+    <td>(Optional) Type of Kubernetes secret. For example, docker-registry or generic. default: docker-registry</td>
+  </tr>
+  <tr>
+    <td><code>secret-name</code><br/>secret name</td>
+    <td>(Only required when creating a new secret) Name of the secret. You can use this secret name in the Kubernetes YAML configuration file. <b>No secret will be created if left unspecified</b>. </td>
+  </tr>
+  <tr>
+    <td><code>arguments</code><br/>arguments</td>
+    <td>(Optional) Specify keys and literal values to insert in generic type secret.For example, --from-literal=key1=value1 --from-literal=key2="top secret".</td>
+  </tr>
+ </table>
+ 
+### Inputs specific to Azure Kubernetes Cluster deploy
+ 
+ <table>
+  <thead>
+    <tr>
+      <th>Action inputs</th>
+      <th>Description</th>
+    </tr>
+  </thead>
+
+  <tr>
+    <td><code>manifests</code><br/>Manifests</td>
+    <td>(Required) Path to the manifest files to be used for deployment</td>
+  </tr>
+  <tr>
+    <td><code>images</code><br/>Images</td>
+    <td>(Optional) Fully qualified resource URL of the image(s) to be used for substitutions on the manifest files. This multiline input accepts specifying multiple artifact substitutions in newline separated form. For example - <br>images: |<br>&nbsp&nbspcontosodemo.azurecr.io/foo:test1<br>&nbsp&nbspcontosodemo.azurecr.io/bar:test2<br>In this example, all references to contosodemo.azurecr.io/foo and contosodemo.azurecr.io/bar are searched for in the image field of the input manifest files. For the matches found, the tags test1 and test2 are substituted.</td>
+  </tr>
+  <tr>
+    <td><code>imagepullsecrets</code><br/>Image pull secrets</td>
```

Comment:
Remove the Extra description of the code if not required. Put the extra description in brackets.

Reply:
I tried to minimize the description as much as possible for all inputs. Please review the changes.